tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+minversion = 2.1
+envlist = py35, py36, py37, lint
+skipsdist = True
+
+[testenv]
+install_command = pip install -U {opts} {packages}
+setenv =
+  VIRTUAL_ENV={envdir}
+  LANGUAGE=en_US
+  LC_ALL=en_US.utf-8
+deps =
+  -r requirements-dev.txt
+  git+https://github.com/Qiskit/qiskit-terra.git
+commands =
+    python setup.py bdist_wheel -- -- -j4
```

Comment:
Not quite sure if `-j4` would fail on Windows, anyway leave it there and I'll change it if I need to.